### PR TITLE
gateway api: ignore error on install CRDs

### PR DESCRIPTION
**Pull request by howardjohn**
The *expected* result is an error, so ignore it.,

Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [ ] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

**Comment by kfaseela** [comment]
you might have to regenerate the snips for test with make gen

**Comment by howardjohn** [comment]
> snips

I just did but only see 1 update. I expect more so I am surprise. Will see if tests pass

**Review by linsun — APPROVED** [review]
Thanks!  LGTM, and tried it too!

**Review by kfaseela — APPROVED** [review]

**Comment by ericvn** [comment]
Thanks @howardjohn. I was Slacking with Frank about this earlier while working through the community tests.

File: content/en/boilerplates/gateway-api-install-crds.md [hunk 1]
```diff
@@ -4,6 +4,6 @@ Note that the Kubernetes Gateway API CRDs do not come installed by default on mo
 installed before using the Gateway API:
 
 {{< text syntax=bash snip_id=install_crds >}}
-$ kubectl get crd gateways.gateway.networking.k8s.io || \
+$ kubectl get crd gateways.gateway.networking.k8s.io &> /dev/null || \
```
[the remark this answers is not in the document]
Comment (ericvn) [reply content/en/boilerplates/gateway-api-install-crds.md:7]: Diff between on my machine:
```
>  kubectl get crd gateways.gateway.networking.k8s.io &> /dev/null || \
   { kubectl kustomize "github.com/kubernetes-sigs/gateway-api/config/crd?ref=v0.6.1" | kubectl apply -f -; }

> kubectl get crd gateways.gateway.networking.k8s.io 2> /dev/null || \ 
   { kubectl kustomize "github.com/kubernetes-sigs/gateway-api/config/crd?ref=v0.6.1" | kubectl apply -f -; }
NAME                                 CREATED AT
gateways.gateway.networking.k8s.io   2023-02-14T16:31:06Z
```

**Comment by frankbu** [comment]
Although the boilerplate covers the majority of the docs that use this command, there are a couple of other ones that have slightly different commands, but do the same `get || create` pattern. We should probably suppress the error in those ones too?

**Pull request merged**